pages/docs/reference/javascript-dce.md

```diff
@@ -5,95 +5,59 @@ category: "JavaScript"
 title: "JavaScript DCE"
 ---
 
-# JavaScript DCE
+# JavaScript Dead Code Elimination (DCE)
```

Comment:
Could we provide information about known issue with dce+ktor?
I mean https://youtrack.jetbrains.com/issue/KT-36484. At least to provide WA explicitly here